htdocs/fichinter/list.php

```diff
@@ -288,7 +288,7 @@
 	$sql .= " AND s.rowid = sc.fk_soc AND sc.fk_user = ".$user->id;
 }
 if ($socid) {
-	$sql .= " AND s.rowid = ".((int) $socid);
+	$sql .= " AND s.rowid = ".$socid;
```

Comment:
Why removing the cast (it is a protection we should have when forging a sql with a test = on a non string data.

Reply:
Hi @eldy
 it's a mistake, I will restore the cast on tomorrow ;)

Reply:
@eldy it's ok now